src/instructlab/configuration.py

```diff
@@ -260,9 +260,9 @@ class _generate(BaseModel):
     num_cpus: PositiveInt = DEFAULTS.NUM_CPUS
     chunk_word_count: PositiveInt = DEFAULTS.CHUNK_WORD_COUNT
     num_instructions: PositiveInt = DEFAULTS.NUM_INSTRUCTIONS
-    output_dir: StrictStr = Field(default_factory=lambda: DEFAULTS.DATASETS_DIR)
-    prompt_file: StrictStr = Field(default_factory=lambda: DEFAULTS.PROMPT_FILE)
-    seed_file: StrictStr = Field(default_factory=lambda: DEFAULTS.SEED_FILE)
+    output_dir: StrictStr = Field(default_factory=DEFAULTS.DATASETS_DIR)
```

Comment:
You can remove `Field` since this was only being used for the lambda functions.

```suggestion
    output_dir: StrictStr = DEFAULTS.DATASETS_DIR
```